core/templates/base-components/footer-donate-volunteer.component.html

```diff
@@ -0,0 +1,25 @@
+<p class="footer-donate-volunteer"
+    [innerHTML]="'I18N_FOOTER_OPPIA_MISSION_DONATE_VOLUNTEER' | translate">
+</p>
+<style>
+  p.footer-donate-volunteer,
+  p.footer-donate-volunteer a {
+    font-size: 24px;
+    font-weight: 400;
+    line-height: 28px;
+    text-transform: none;
+  }
+
+  p.footer-donate-volunteer a {
+    color: #c55f45;
+    text-decoration: underline;
+  }
+
+  @media (max-width: 600px) {
+    p.footer-donate-volunteer,
+    p.footer-donate-volunteer a {
+      font-size: 20px;
+      line-height: 24px;
+    }
+  }
+</style>
```

Comment:
Looks like you are missing a new line here. Can you also get the RTL version checked once with you latest changes?

Reply:
PTAL
![rtl](https://github.com/oppia/oppia/assets/76906722/9339007e-16e1-4003-8d8d-94aacabc095f)